employee_service/src/employee_repository.py

```diff
@@ -6,6 +6,8 @@ class EmpRepository:
     def connect_db():
         return sqlite3.connect(EmpRepository.DBPATH)
     
+
```

Comment:
variables names inside this file can be improved.
Like, on line number 111, the variable name `insert_cursor` is not doing as its name suggest. It can be changed